### ISSUE TITLE
Breadcrumbs inconsistent throughout the site

### ISSUE THREAD

**Issue by svpatel1** [NIHL/Drupal7-Migration#179]
Item #50: Breadcrumbs - Noticing that Training Type and Other Resources are not clickable in breadcrumb. There is no landing page or default page.

We would like a decision made about how to handle the breadcrumb. The choices are:

a. Create a landing page for each of the sub categories - e.g. service->requests, services->library services, resources-> other research tools, etc. OR
 
b. completely remove the intermediate subcategory from the breadcrumb (e.g. requests, library services, other research tools, etc.)

**Comment by theBrarian** [comment]
Will discuss with the group about the options.

**Comment by theBrarian** [comment]
I will bring to group for discussion.

**Comment by mashanadavis** [comment]
Ganga and I spoke about this. We vote for option b. where the subheading/subcategory is removed from the breadcrumb.

**Comment by theBrarian** [comment]
I would agree, but with the caveat of improving the landing pages for the categories.

**Comment by mashanadavis** [comment]
If I understand your comment, a landing page will be created for each global menu option (Home, Resources, Services, Training, About Us, and Log In). Some already exists.

**Comment by theBrarian** [comment]
Yes that is the gist. Have a landing page for the global menu items that are more built out. This is where the breadcrumbs would land. Not to mention the links from the Catalog that would not include the dropdown in the global navigation. They need to include more robust content of what is available within that section.

**Comment by gbohara** [comment]
Can we assume that this can be done by NIH (creating landing pages for certain menu items)?

**Comment by theBrarian** [comment]
Yes. We will work on these pages.

**Comment by alwaysmashana** [comment]
Moving back to Ongoing Discussion until we settle this internally. 

I'd rather create landing pages for the global navigation options vs. these subheading menu options. We do not currently have this on SharePoint.

**Comment by alwaysmashana** [comment]
http://nihlibrary.prod.acquia-sites.com/services/bioinformatics-consult
Fix breadcrumb for this. It is showing 3rd level item.

**Comment by svpatel1** [comment]
We believe that the breadcrumb working is working properly (or at least as designed). Please note that when you go to Services-> Bioinformatics consult from the services menu, you go to a child of 'Bioinformatics Support Program", therefore "Bioinformatics Support Program" shows up in the breadcrumb as the parent.  This is consistent with how all of the other pages work - i.e. only the parent of the page that you are on, will show up in the breadcrumb.  Let us know if you would like us to walk you thru it.
